packages/jobs/lib/execution/action.ts

```diff
@@ -47,6 +51,11 @@ export async function startAction(task: TaskAction): Promise<Result<void>> {
             throw new Error(`Action config not found: ${task.id}`);
         }
 
+        const getEndUser = await getEndUserByConnectionId(db.knex, { connectionId: task.connection.id });
+        endUser = getEndUser.isOk()
+            ? { id: getEndUser.value.id, endUserId: getEndUser.value.endUserId, orgId: getEndUser.value.organization?.organizationId || null }
+            : null;
```

Comment:
`endUser` is already null so why not `if(getEndUser.isOk()) { endUser = ... }`

Reply:
indeed